### ISSUE TITLE
Feature request: let IE8 use one specific responsive viewport

### ISSUE THREAD

**Issue by fchristant** [scottjehl/Respond#171]
This is just me thinking out loud, so bear with me...

I'm working on a complex mobile-first responsive design, and of course I came across the IE8 issue applying only the default (mobile) styles. In my search for a solution I came across "Respond" and it instantly fixed so many issues, I'm very grateful for this solution.

However, my original goal of fixing my design to be usable in IE8 was to offer it only the largest responsive mode, in my case the design that applies as of 1024px of width. That design is basically the result of all lower viewport styles combined. 

In other words, I do not really want IE8 to be responsive. It's a desktop browser and I want to always give it the desktop styles. I can do this somewhat using a CSS preprocessor, by creating a seperate SCSS file for IE8 and below, which imports all style modules, but I was wondering if someway, somehow, Respond could be equipped with a feature to let IE8 use one viewport mode exclusively.

Does that request make sense?
